index.html

```diff
@@ -1263,6 +1273,61 @@ <h4>
             </li>
           </ol>
         </section>
+        <section data-dfn-for="HIDDeviceExclude">
+          <h4>
+            <dfn>HIDDeviceExclude</dfn> dictionary
```

Comment:
I think there's not much value in defining a new dictionary type for exclusion filters. Let's reuse `HIDDeviceFilter` for the exclusion filters and specify that `usagePage` and `usage` must not be defined when a `HIDDeviceFilter` is used for exclusion.

Reply:
Done.

Reply:
Actually, I think it makes more sense to treat exclusion filters exactly the same as normal (inclusion?) filters other than whether they include or exclude. If the filters look the same they should behave the same. Let's allow filtering by `usagePage` and `usage` (even though I can't think of a good reason to do that). In the future I think we may add filter parameters for connection type (usb, bluetooth, etc) and it would be nice to not need duplicate language in the spec for inclusion vs exclusion.

Reply:
I've updated spec PR based on your feedback. Please have another look.